src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -682,26 +693,29 @@ protected DynamoModel(IStartConfiguration config)
             }
 
             bool areAnalyticsDisabledFromConfig = false;
-            try
-            {
-                // Dynamo, behind a proxy server, has been known to have issues loading the Analytics binaries.
-                // Using the "DisableAnalytics" configuration setting, a user can skip loading analytics binaries altogether.
-                var assemblyConfig = ConfigurationManager.OpenExeConfiguration(GetType().Assembly.Location);
-                if (assemblyConfig != null)
+            if (!IsServiceMode)
+            {   // Skip getting the value for areAnalyticsDisabledFromConfig because analytics is disabled for searvice mode anyway
+                try
                 {
-                    var disableAnalyticsValue = assemblyConfig.AppSettings.Settings["DisableAnalytics"];
-                    if (disableAnalyticsValue != null)
-                        bool.TryParse(disableAnalyticsValue.Value, out areAnalyticsDisabledFromConfig);
+                    // Dynamo, behind a proxy server, has been known to have issues loading the Analytics binaries.
+                    // Using the "DisableAnalytics" configuration setting, a user can skip loading analytics binaries altogether.
+                    var assemblyConfig = ConfigurationManager.OpenExeConfiguration(GetType().Assembly.Location);
+                    if (assemblyConfig != null)
+                    {
+                        var disableAnalyticsValue = assemblyConfig.AppSettings.Settings["DisableAnalytics"];
+                        if (disableAnalyticsValue != null)
+                            bool.TryParse(disableAnalyticsValue.Value, out areAnalyticsDisabledFromConfig);
+                    }
+                }
+                catch (Exception)
+                {
+                    // Do nothing for now
                 }
-            }
-            catch (Exception)
-            {
-                // Do nothing for now
             }
 
             // If user skipped analytics from assembly config, do not try to launch the analytics client
-            // or the feature flags client.
-            if (!areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics)
+            // or the feature flags client for web traffic reason.
+            if (!areAnalyticsDisabledFromConfig && !Dynamo.Logging.Analytics.DisableAnalytics && !IsServiceMode)
```

Comment:
We can put the IsServiceMode check at the beginning of these conditions.

Reply:
Will do